test_runner/random_test_runner/include/random_test_runner/lanelet_utils.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2015-2020 Tier IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Developed by Robotec.ai.
+
+#ifndef RANDOM_TEST_RUNNER__LANELET_UTILS_HPP
+#define RANDOM_TEST_RUNNER__LANELET_UTILS_HPP
+
+#include <lanelet2_core/LaneletMap.h>
+#include <lanelet2_routing/RoutingGraph.h>
+
+#include <boost/filesystem.hpp>
+
+#include "geometry_msgs/msg/pose_stamped.hpp"
+#include "random_test_runner/data_types.hpp"
+
+namespace hdmap_utils
+{
+class HdMapUtils;
+}
+
+struct LaneletPart
+{
+  int64_t lanelet_id;
+  double start_s;
+  double end_s;
+};
+
+class LaneletUtils
+{
+public:
+  LaneletUtils(const boost::filesystem::path & filename);
```

Comment:
[Boost should be avoided unless absolutely required.](https://docs.ros.org/en/galactic/Contributing/Code-Style-Language-Versions.html?highlight=boost#boost)
In C++14 the only alternative is `std::experimental::filesystem` (or plain string), but I don't know if it's worth